pkg/shell/terraform.go

```diff
@@ -136,9 +148,29 @@ func getOutputs(tf *tfexec.Terraform) (map[string]cty.Value, error) {
 		}
 	}
 
+	var apply bool
 	if wantsChange {
-		return map[string]cty.Value{},
-			fmt.Errorf("cloud infrastructure requires changes; please run \"terraform -chdir=%s apply\"", tf.WorkingDir())
+		log.Println("cloud infrastructure requires changes")
```

Comment:
nit: Remove timestamp from output lines. If you do not do this in this PR, please open a task.